packages/ui-number-input/src/NumberInput/styles.css

```diff
@@ -1,138 +0,0 @@
-/* stylelint-disable selector-max-class */
-.inputWidth {
-  display: block;
-  position: relative;
-
-  &::before {
-    content: "";
-    pointer-events: none;
-    box-sizing: border-box;
-    display: block;
-    position: absolute;
-    top: -0.25rem;
-    bottom: -0.25rem;
-    left: -0.25rem; /* stylelint-disable-line property-blacklist */
-    right: -0.25rem; /* stylelint-disable-line property-blacklist */
-    border: var(--focusOutlineWidth) var(--focusOutlineStyle) var(--focusOutlineColor);
-    border-radius: calc(var(--borderRadius) * 1.5);
-    transition: all 0.2s;
-
-    /* properties to transition on :focus */
-    opacity: 0;
-    transform: scale(0.95);
-  }
-
-  &.focus::before {
-    opacity: 1;
-    transform: scale(1);
-  }
-
-  &.focus.invalid::before {
-    border-color: var(--errorOutlineColor);
-  }
-}
-
-.inputContainer {
-  display: flex;
-  margin: 0;
-  box-sizing: border-box;
-  transition: all 0.2s;
-  overflow: hidden;
-  font-family: var(--fontFamily);
-  border: var(--borderWidth) var(--borderStyle) var(--borderColor);
-  border-radius: var(--borderRadius);
-
-  &.invalid {
-    border-color: var(--errorBorderColor);
-  }
-
-  &.disabled {
-    cursor: not-allowed;
-    pointer-events: none;
-    opacity: 0.5;
-  }
-
-  &.medium {
-    font-size: var(--mediumFontSize);
-    height: var(--mediumHeight);
-  }
-
-  &.large {
-    font-size: var(--largeFontSize);
-    height: var(--largeHeight);
-  }
-}
-
-/* stylelint-disable selector-max-type, no-descending-specificity */
-
-.input,
-input[type].input {
```

Comment:
I'm just curious, what purpose did this serve? Did it need something to migrate to the new code?

Reply:
I think with the restructuring of the styling, this line is implied in the style.